### PR TITLE
Move selected line up and down in Linux

### PR DESCRIPTION
**Pull request by israelcrux**
'ctrl-shift-up': 'editor:move-line-up'
'ctrl-shift-down': 'editor:move-line-down'
For some reason this is default for OSX but not for Linux, while Sublime's default configuration for this keymap in Linux and Windows is indeed moving the line up or down

### CODE REVIEW

**Comment by mehcode** [comment]
https://github.com/dantaex/atom/blob/patch-2/keymaps/linux.cson#L115-L116

File: keymaps/linux.cson [hunk 1]
```diff
@@ -51,8 +51,8 @@
   'ctrl-shift-tab': 'pane:show-previous-item'
   'ctrl-pageup': 'pane:show-previous-item'
   'ctrl-pagedown': 'pane:show-next-item'
-  'ctrl-shift-up': 'core:move-up'
```
Comment (mnquintana) [inline keymaps/linux.cson]: Would you mind re-mapping https://github.com/dantaex/atom/blob/patch-2/keymaps/linux.cson#L115-L116 to `core:move-up`?

**Comment by Lomanic** [comment]
How to get these shortcuts working  (on Linux) in `~/.atom/keymap.cson`?

The following does not work:

``` coffeescript
'atom-workspace atom-text-editor:not([mini])':
  'ctrl-up': 'core:move-up'
  'ctrl-down': 'core:move-down'
  'ctrl-shift-up': 'editor:move-line-up'
  'ctrl-shift-down': 'editor:move-line-down'
```

**Comment by damieng** [comment]
PR is out of sync and hasn't been merged in 2 years.

**Pull request closed without merging**